Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1065 @@
+}
+```
+
+---
+
+### 내 프로필 조회
+
+**GET** `/users/{nickname}`
+
+로그인된 닉네임에 해당하는 사용자 정보를 조회합니다. 단 비밀번호는 제외하고 조회합니다.
+
+**Path Parameters**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)